packages/shared/src/store/sync.ts

```diff
@@ -838,20 +837,26 @@ async function handleGroupUpdate(update: api.GroupUpdate, ctx: QueryCtx) {
       break;
     case 'moveChannel':
       logger.log('moving channel', update);
-      await moveChannel({
+      // eslint-disable-next-line
```

Comment:
why do we need eslint disable here? If it's the variable thing, should we fetch the group in the action instead?